habitat/tasks/nav/greedy_follower.py

```diff
@@ -0,0 +1,210 @@
+from typing import Union
```

Comment:
Regarding `greedy_follower.py` name, it can be misleading as it doesn't state shortest path anywhere and we have agents that are followers (Goal Follower) as well., maybe something like `shortest_path_follower.py`

Reply:
Done